src/status_im/contexts/communities/actions/channel_view_details/view.cljs

```diff
@@ -6,32 +6,44 @@
+            [status-im.constants :as constants]
             [status-im.contexts.communities.actions.channel-view-details.style :as style]
             [status-im.feature-flags :as ff]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
 (defn- contact-item
-  [public-key]
-  (let [show-profile-actions                         #(rf/dispatch
-                                                       [:show-bottom-sheet
-                                                        {:content (fn [] [home.actions/contact-actions
-                                                                          {:public-key public-key}])}])
-        [primary-name secondary-name]                (rf/sub [:contacts/contact-two-names-by-identity
-                                                              public-key])
-        {:keys [ens-verified added? compressed-key]} (rf/sub [:contacts/contact-by-address public-key])
-        theme                                        (quo.theme/use-theme)]
+  [public-key _ _ {:keys [theme]}]
+  (let [show-profile-actions          (rn/use-callback
+                                       (fn []
+                                         (rf/dispatch
+                                          [:show-bottom-sheet
+                                           {:content (fn [] [home.actions/contact-actions
+                                                             {:public-key
+                                                              public-key}])}]))
+                                       [public-key])
+        on-press                      (rn/use-callback
+                                       (fn []
+                                         (rf/dispatch [:chat.ui/show-profile public-key]))
+                                       [public-key])
+        [primary-name secondary-name] (rf/sub [:contacts/contact-two-names-by-identity public-key])
+        {:keys [ens-verified
+                added?
+                compressed-key]}      (rf/sub [:contacts/contact-by-address public-key])]
     [contact-list-item/contact-list-item
-     {:on-press      #(rf/dispatch [:chat.ui/show-profile public-key])
+     {:on-press      on-press
       :on-long-press show-profile-actions
       :accessory     {:type     :options
                       :on-press show-profile-actions}}
-     {:primary-name   primary-name
-      :secondary-name secondary-name
-      :compressed-key compressed-key
-      :public-key     public-key
-      :ens-verified   ens-verified
-      :added?         added?}
+     {:primary-name    primary-name
+      :secondary-name  secondary-name
+      :compressed-key  compressed-key
+      :public-key      public-key
+      :ens-verified    ens-verified
+      :added?          added?
+      ;; We hardcode the height of the container to match exactly the height
+      ;; used in the `get-item-layout` function.
+      :container-style {:height constants/contact-item-height}}
```

Comment:
As you can see in [Figma](https://www.figma.com/design/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=114-6475&t=zT2O1qoUKavCY2Fq-0), every user item component (the one used in the members' list) has a height of 56, except for a variant that can have an extra message which increases the height. This variant is not implemented by us yet and, in fact, it's something I believe we should never implement because this component can be displayed by the hundreds in the members' list and we really want the optimization provided by `getItemLayout`, which requires a fixed height. I left a comment in the component as a reminder to a future developer.